.vib/cert-manager-webhook/goss/cert-manager-webhook.yaml

```diff
@@ -0,0 +1,5 @@
+command:
+  # There are no syntax error in the configuration
+  check-cert-manager-webhook:
+    exec: /opt/bitnami/cert-manager-webhook/bin/cert-manager-webhook -h
```

Comment:
Same feedback from https://github.com/bitnami/containers/pull/34256.

Please remove the wrong comment and replace the command to not use its full path.